version: "3.9"

services:
  voice_assistant:
    build:
      context: .
#      args:
#        USER_ID: ${USER_ID:-0}
#        GROUP_ID: ${GROUP_ID:-0}
    stdin_open: true
    tty: true
    volumes:
      - ./:/app
      - ./asound.conf:/etc/asound.conf
    devices:
      - /dev/snd:/dev/snd
    environment:
      - LANG=C.UTF-8
      - PYTHONDONTWRITEBYTECODE=1
      - PYTHONUNBUFFERED=1
      - OPENWEATHER_API_KEY=${OPENWEATHER_API_KEY}
    networks:
      - testing
  rhvoice_rest:
    image: aculeasis/rhvoice-rest:amd64
    ports:
      - '8090:8080'
    depends_on:
      - voice_assistant
    networks:
      - testing

networks:
  testing:
    external: true
